product.8.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
-		<meta charset="utf-8">
+		<meta charset="utf-9">
```

Comment:
Please review the changes which are there due to using the replaceall feature. There are some unnecessary changes. This won't happen if you review the changes made before committing them.